tests/utils/normalizeSnapshot.ts

```diff
@@ -13,5 +13,15 @@ export function normalizeSnapshot(html: string) {
       .replace(/disabled="disabled"/gs, 'disabled=""')
       // Vue renders a useless extra space between last list item element and closing list tag
       .replace(/<\/li> <\/ul>/g, '</li></ul>')
+      // Vue renders extra whitespace between span elements
+      .replace(/<\/span> <span/g, '</span><span')
+      // Vue renders an empty comment for falsy v-if predicates
+      .replace(/(\s+)?<!---->/g, '') // Vue 2
+      .replace(/(\s+)?<!--v-if-->/g, '') // Vue 3
+      // Vue renders extra whitespace after list elements
+      .replace(/<\/ul> </g, '</ul><')
+      // InstantSearch.js <Template> adds a div wrapper around menu items
+      // @MAJOR: Remove this once <Template> stops defaulting to div root tags
+      .replace(/<div>\s*(<a[\s\S]*?<\/a>)\s*<\/div>/g, '$1')
```

Comment:
Wondering if I should scope this string replace to only the common menu test runs.

Currently it doesn't affect other tests relying on this normalize function, but it could create issues in the future with other widgets?

Reply:
Agreed, this seems to broad, and should be for the menu test alone

Reply:
You can scope it to `<Menu>` with the following regex:

```txt
/<div>\s*(<a class="ais-Menu-link".+?><\/a>)\s*<\/div>/g
```

Reply:
I did both, I better scoped it to only match `ais-Menu-link` and moved it into the options test suite for the menu widget. See https://github.com/algolia/instantsearch/pull/5825/commits/6e06ebc71dfb52836b445cc874e639c63d2b37e7.